islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -147,6 +148,15 @@
       <xsl:with-param name="pid" select="$pid"/>
       <xsl:with-param name="datastream" select="$datastream"/>
     </xsl:call-template>
+
+    <!-- Potential qualified date range for dateCreated. -->
+    <xsl:if test="mods:dateCreated">
+      <xsl:call-template name="qualified_date_range">
+        <xsl:with-param name="prefix" select="$prefix"/>
+        <xsl:with-param name="suffix" select="$suffix"/>
+        <xsl:with-param name="value" select="normalize-space(mods:dateCreated/text())"/>
+      </xsl:call-template>
+    </xsl:if>
```

Comment:
Yeah I looked it up and apparently you can indeed have multiple dateCreated-s - which makes sense if you're using them with attributes I guess